src/AppBundle/Installer/AppInstaller.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace AppBundle\Installer;
+
+use EzSystems\PlatformInstallerBundle\Installer\CleanInstaller;
+use EzSystems\PlatformInstallerBundle\Installer\Installer;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Install data from ezplatform.com
+ */
+class AppInstaller extends CleanInstaller  implements Installer
+{
+    public function importData()
+    {
+        $this->runQueriesFromFile('app/Resources/installer/sql/app_data.sql');
```

Comment:
Nope, this have to be downloaded by the user first `https://ezplatform.com/dumps/ezplatform_page_db.sql.bz2`